.github/workflows/ci.yaml

```diff
@@ -20,11 +20,11 @@ jobs:
       - name: "Update Homebrew"
         run: brew update
 
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - name: "Install local casks with Homebrew"
         run: |
-          mkdir -p /usr/local/Homebrew/Library/Taps/guardian
-          ln -s $PWD /usr/local/Homebrew/Library/Taps/guardian/homebrew-devtools
+          mkdir -p $(brew --repository)/Library/Taps/guardian
```

Comment:
The location Homebrew installs taps to in macOS Somona has changed. Using `brew --repository` is hopefully a bit more future-proof.